packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/translations/Messages_fr_FR.json

```diff
@@ -23,6 +23,7 @@
   "data_processing_details_configuration_python_file_label": "Fichier Python",
   "data_processing_details_configuration_main_class_label": "Classe principale",
   "data_processing_details_configuration_arguments_label": "Arguments",
+  "data_processing_details_configuration_ttl_label": "Time To Live (TTL)",
```

Comment:
**question:** Are we expecting any translation based on the initial content?

If not, then we are relying on static content that can be stored as a constant or directly added into the template (if possible).

Reply:
Removed from translation files and added directly to the template here: https://github.com/ovh/manager/pull/7116/commits/82042c2a0a5a9e5d0eefe3656d93bd7ead21eea9